# SCA-seq
This repository contains all the code that was used to analyze SCA-seq data from raw Nanopore data (both fast5 and fastq) to the tabular data.
At the start of each file, dependencies like essential libraries or fixed paths to software and data files are declared, which are required for the scripts to function properly.

The detailed output format is as follows:

Step3_single_mol_methy.sh

${sample}_single_mol_methy_seg.xls
Tabular data that links segment accessiblity and CpG ratio to segment genomic locus.
![image](https://user-images.githubusercontent.com/91398995/226509102-2071014d-0dd6-47cc-9b25-0ade511dff46.png)

${sample}_single_mol_methy_site.xls
Tabular data of GpC/CpG methylation likelihood ratio for each GpC/CpG site.
![image](https://user-images.githubusercontent.com/91398995/226511394-7c864285-04ee-4b7d-92e1-b0d08ccbe841.png)


Step2_hic.sh

SCA-seq concatemers are converted to the pairwise contact and .hic file was generated by juicer_tools
${sample}.hic



